solr/solr-ref-guide/modules/query-guide/pages/dense-vector-search.adoc

```diff
@@ -279,13 +279,12 @@ The `knn` query parser can be used with filter queries:
 
 [IMPORTANT]
 ====
-When using `knn` in these scenarios make sure you have a clear understanding of how filter queries work in Apache Solr:
 
-The Ranked List of document IDs resulting from the main query `q` is intersected with the set of document IDs deriving from each filter query `fq`.
+Filter queries are executed as pre-filtering for the main query. This means that in the first scenario, the query id:(1,2,3)
```

Comment:
This documentation change is not clear:
Maybe you meant:
"
Filter queries are executed as pre-filters: the main query refines the sub-set of search results derived from the application of all the filter queries combined as 'MUST' clauses(boolean AND).

This means that in 
[source,text]
&q=id:(1 2 3)&fq={!knn f=vector topK=10}[1.0, 2.0, 3.0, 4.0]
The results are prefiltered by the topK knn retrieval and then only the documents from this subset, matching the query 'q=id:(1 2 3)' are returned.

In 
[source,text]
&q={!knn f=vector topK=10}[1.0, 2.0, 3.0, 4.0]&fq=id:(1 2 3)
The results are prefiltered by the fq=id:(1 2 3) and then only the documents from this subset are considered as candidates for the topK knn retrieval.



"